Fix up all gometalinter issues

```
stopwatch git:(master) gometalinter .
lap.go:9:6:warning: exported type Lap should have comment or be unexported (golint)
stopwatch.go:34:1:warning: exported method Stopwatch.SetFormatter should have comment or be unexported (golint)
stopwatch.go:40:1:warning: exported method Stopwatch.MarshalJSON should have comment or be unexported (golint)
version.go:3:7:warning: exported const VERSION should have comment or be unexported (golint)
```

Also, should add GoReportCard and GoDoc badges to README.

Some Gofmt issues need to be fixed according to the GoReportCard.

--- a/README.md
+++ b/README.md
@@ -4,6 +4,8 @@ stopwatch
 This package offers a nice solution to take some measurements of the various states of your application.  It is a non high-resolution timer that is designed to be fast giving you an accurate picture of how long your code paths are taking.
 
 [![BuildStatus](https://travis-ci.org/sendgrid/stopwatch.svg?branch=master)](https://travis-ci.org/sendgrid/stopwatch)
+[![Go Report Card](https://goreportcard.com/badge/github.com/sendgrid/stopwatch)](https://goreportcard.com/report/github.com/sendgrid/stopwatch)
+[![GoDoc](https://godoc.org/github.com/sendgrid/stopwatch?status.svg)](https://godoc.org/github.com/sendgrid/stopwatch)
 
 ### Usage
 

--- a/lap.go
+++ b/lap.go
@@ -6,6 +6,7 @@ import (
 	"time"
 )
 
+// Lap represents a split time from the stopwatch
 type Lap struct {
 	formatter func(time.Duration) string
 	state     string

--- a/stopwatch.go
+++ b/stopwatch.go
@@ -31,12 +31,14 @@ func New(offset time.Duration, active bool) *Stopwatch {
 	return &sw
 }
 
+// SetFormatter takes a function that converts time.Duration into a string
 func (s *Stopwatch) SetFormatter(formatter func(time.Duration) string) {
 	s.Lock()
 	s.formatter = formatter
 	s.Unlock()
 }
 
+// MarshalJSON converts into a slice of bytes
 func (s *Stopwatch) MarshalJSON() ([]byte, error) {
 	return []byte(s.String()), nil
 }

--- a/stopwatch_test.go
+++ b/stopwatch_test.go
@@ -108,11 +108,11 @@ func TestMultiThreadLaps(t *testing.T) {
 	sw.Stop()
 
 	expected := map[string]struct{}{
-		"Create File": struct{}{},
-		"task 0":      struct{}{},
-		"task 1":      struct{}{},
-		"Upload File": struct{}{},
-		"task A":      struct{}{},
+		"Create File": {},
+		"task 0":      {},
+		"task 1":      {},
+		"Upload File": {},
+		"task A":      {},
 	}
 
 	laps := sw.Laps()

--- a/version.go
+++ b/version.go
@@ -1,3 +1,4 @@
 package stopwatch
 
+// VERSION represents the version of this library
 const VERSION = "1.0.0"

